pkg/storaged/stratis-details.jsx

```diff
@@ -726,13 +937,28 @@ const StratisStoppedPoolSidebar = ({ client, uuid }) => {
 };
 
 export const StratisStoppedPoolDetails = ({ client, uuid }) => {
+    const stopped_props = client.stratis_manager.StoppedPools[uuid];
+    const keydesc_info = stopped_props.key_description;
+    const clevis_info = stopped_props.clevis_info;
+
+    const encrypted = (keydesc_info && keydesc_info.v[0]) || (clevis_info && clevis_info.v[0]);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230602-113720-366c6399-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)